gtk/gtk.go

```diff
@@ -9065,6 +9066,21 @@ func (v *TreeModelFilter) SetVisibleColumn(column int) {
 	C.gtk_tree_model_filter_set_visible_column(v.native(), C.gint(column))
 }
 
+// ConvertChildPathToPath is a wrapper around gtk_tree_model_filter_convert_child_path_to_path().
+func (v *TreeModelFilter) ConvertChildPathToPath(childPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_filter_convert_child_path_to_path(v.native(), childPath.native())
+	if path == nil {
+		return nil
+	}
+	p := &TreePath{path}
+	return p
+}
+
+// ConvertChildIterToIter is a wrapper around gtk_tree_model_filter_convert_child_iter_to_iter().
+func (v *TreeModelFilter) ConvertChildIterToIter(sortIter, childIter *TreeIter) bool {
+	return gobool(C.gtk_tree_model_filter_convert_child_iter_to_iter(v.native(), sortIter.native(), childIter.native()))
+}
```

Comment:
ConvertChildIterToIter() should have only one input parameter and two return values (in the spirit of golang).

Furthermore, sortIter (please rename it to `filterIter`) may be nil according to [documentation](https://developer.gnome.org/gtk3/unstable/GtkTreeModelFilter.html#gtk-tree-model-filter-convert-child-iter-to-iter) and will therefore crash when you call it

Reply:
Calls to nil are OK with golang in this case.
See definition of [TreeIter.native()](https://github.com/gotk3/gotk3/blob/69caac910e503eb4c7d2de1c82f36de61dca361a/gtk/gtk.go#L8774).
However, this should never be a nil value - just an uninitialized iter, meaning a valid pointer, just default content. (`new(gtk.TreeIter)`)

filterIter is an out variable so one could indeed argue that it should be a return value instead.